ubuntu-advantage

```diff
@@ -151,7 +267,7 @@ is_livepatch_enabled() {
     canonical-livepatch status >/dev/null 2>&1
 }
 
-validate_esm_token(){
+validate_esm_or_fips_token(){
```

Comment:
I'd rename this to `validate_user_pass_token`, so it's more explicit

Reply:
ok